tests/utils/test_story.py

```diff
@@ -33,3 +39,45 @@ def test_get_siblings_description(display_name, label, backward, expected):
     rv = story_utils.get_siblings_description(
         display_name, ContainerStoryManager().story_flow(label), backward)
     assert rv == expected
+
+
+def test_get_recent_nodes():
+    """Test the get_recent_nodes function."""
+    BugzillaBug.get_or_create({
+        'id_': '11111',
+        'modified_time': datetime(2017, 4, 26, 11, 44, 38)
+    })[0]
```

Comment:
`[0]` is unnecessary since you aren't storing the data in a variable. This applies for the remainder of the test.